modules/mono/mono_gd/support/android_support.cpp

```diff
@@ -39,9 +39,10 @@
 
 #include "core/os/os.h"
 #include "core/string/ustring.h"
-#include "platform/android/java_godot_wrapper.h"
-#include "platform/android/os_android.h"
-#include "platform/android/thread_jandroid.h"
+
+#include "java_godot_wrapper.h"
+#include "os_android.h"
+#include "thread_jandroid.h"
```

Comment:
This change is debatable, it's making it consistent with `modules/openxr/extensions/openxr_android_extension.cpp` which does the same. Those files are only built on Android, and thus `platform/android` is in the include path.

Making those relative to `platform/android` as if they were part of the `platform/android` modular component means that if the module could be renamed / moved without hassle.

Reply:
I don't get it. Old paths don't affect our ability to rename or move this module, since they are relative to the overall project root. I'd prefer them.

Reply:
After this change, the only hardcoded `platform/` includes are in the unmaintained UWP code:
```
$ rg "include .platform/.*/"
platform/uwp/os_uwp.cpp
42:#include "platform/windows/windows_terminal_logger.h"

platform/uwp/app_uwp.cpp
37:#include "platform/windows/key_mapping_windows.h"

editor/SCsub
36:        reg_exporters_inc += '#include "platform/' + e + '/export/export.h"\n'
```

Reply:
I see!